.github/workflows/branch-enforcer.yml

```
@@ -3,6 +3,8 @@ name: External PR Branch Enforcer
 on:
   pull_request_target:
     types: [opened, edited, synchronize, reopened]
+  pull_request_target:
```

actionlint (1.7.11)

[error] key "pull_request_target" is duplicated in "on" section. previously defined at line:4,col:3

(syntax-check)